CODEOWNERS

```diff
@@ -105,6 +105,9 @@
 # The `logging` subchart
 /resources/logging/ @rakesh-garimella @joek @sayanh @venturasr @lilitgh @k15r
 
+# The `log-ui` subchart
+/resources/logging/charts/log-ui/ @rakesh-garimella @joek @sayanh @venturasr @lilitgh @k15r @pekura @maxmarkus @jesusreal @kwiatekus @dariadomagala @hardl @y-kkamil
```

Comment:
```suggestion
/resources/logging/charts/log-ui/ @pekura @maxmarkus @jesusreal @kwiatekus @dariadomagala @hardl @y-kkamil
```

Reply:
i think automix is not needed here

Reply:
I spoke to @a-thaler and we came up we the final list.